### PR TITLE
Add switch for each executable not to compile unnecessary executables

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES:
- Add the switches not to compile each executable to `CMakeLists.txt`.
- Rename the source directories (add the suffix '.fd' for the fortran source codes) to meet the NCO standards.

## TESTS CONDUCTED:
Build test in the RRFS workflow (EMC)

## ISSUE (optional):
Fixes issue mentioned in #55

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -56,26 +79,70 @@ find_package(gsi REQUIRED)
...
+endif()
+if(UPDATE_SNOW)
+  add_subdirectory(update_snow.fd)
+endif()
+if(UPDATE_SST)
+  add_subdirectory(update_sst.fd)
+endif()
+if(USE_RAPHRRR_SFC)
+  add_subdirectory(use_raphrrr_sfc.fd)
+endif()
```
Comment (guoqing-noaa) [inline CMakeLists.txt]: @chan-hoo Thanks a lot for making these changes!! A function we've been waiting for!

I have a comment to add a top-level option such as "RRFS_UTL_ALL" which can turn on/off building all executables except those explicitly set in the cmake command line. 
Changes are in this PR to your fork:
https://github.com/chan-hoo/rrfs_utl/pull/1

Comment (chan-hoo) [reply CMakeLists.txt]: @guoqing-noaa, thanks for the PR. Merged.

**Review by guoqing-noaa — APPROVED** [review]
LGTM

**Comment by guoqing-noaa** [comment]
@hu5970 and @chunhuazhou Please review this PR at your earlier convenience. Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (guoqing-noaa) [reply CMakeLists.txt]: @chan-hoo Thanks for merging the changes!

**Review by hu5970 — APPROVED** [review]

**Pull request merged**